src/followers/render.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Server-side rendering of the followers block.
+ *
+ * @package Activitypub
+ */
+
+use Activitypub\Blocks;
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Followers;
+
+/* @var array $attributes Block attributes. */
+$attributes = wp_parse_args( $attributes );
+
+/* @var string $content Inner blocks content. */
+$content = $content ?? '';
+if ( empty( $content ) ) {
```

Comment:
```suggestion
if ( empty( $content ) ) {
```

why setting to empty and then check for empty?

Reply:
`$content` on its own is an undefined global in the context of `render.php`. The ternary provides a fallback in case it actually is undefined.

Reply:
but that the same for `empty` it also checks implicitly for `isset`.

> No warning is generated if the variable does not exist. That means empty() is essentially the concise equivalent to !isset($var) || $var == false. This also applies to nested structures, such as a multidimensional array or chained properties. 

https://www.php.net/manual/en/function.empty.php

Reply:
There is an else case below, that deals with innerBlocks.

Reply:
sure, but the `else` case is only called if `$content` is not empty?